MobileFilter/sections/specific_web.txt

```diff
@@ -42,6 +42,7 @@ anwap.club,anwap.film,anwap.mobi##body > div[class] > a > strong
 ||mobile.donanimhaber.com/ads^
 !
 ficbook.net##.jsHideStickyRkl
+||etuch.cn^$image,domain=m.silisili.me
```

Comment:
Not requested for me; also getting redirected several times until I hit silisili.cc

Reply:
When I open dev console it always redirects me to another page, but I only reproduced it (banner) once.

Reply:
I don't see ads when Easylist China is enabled. But redirected to another site.